<launch>
    <!-- Define parameters for hand, model, and controller -->
    <arg name="hand" default="left" />
    <arg name="model" default="7" />
    <arg name="controller" default="TorqueController" />

    <!-- Launch the Allegro Hand Controller based on the 'hand' argument -->
    <include file="$(find allegro_hand_controllers)/launch/allegro_hand.launch">
        <arg name="HAND" value="$(arg hand)" />
    </include>

    <!-- Launch the iiwa_driver with specified model and controller -->
    <include file="$(find iiwa_driver)/launch/iiwa_bringup.launch">
        <arg name="model" value="$(arg model)" />
        <arg name="controller" value="$(arg controller)" />
    </include>

</launch>
